inksdk/ink/src/main/java/com/microsoft/device/ink/InkView.kt

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.ink
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.BlendMode
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.graphics.SurfaceTexture
+import android.os.Build
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.Surface
+import android.view.TextureView
+import androidx.annotation.RequiresApi
+
+@RequiresApi(Build.VERSION_CODES.Q)
+class InkView constructor(
+    context: Context,
+    attributeSet: AttributeSet
+) :
+    TextureView(context, attributeSet), TextureView.SurfaceTextureListener {
+
+    private var mSurface: Surface? = null
```

Comment:
we don't user hungarian notation, so please remove the `m` so its `sureface` and we keep consinstency between all other sdk code.